localstack/services/sns/sns_listener.py

```diff
@@ -237,3 +240,58 @@ def get_message_attributes(req_data):
             break
 
     return attributes
+
+
+def evaluate_filter_policy_conditions(conditions, attribute):
+    result = False
+
+    if attribute is None:
+        return False
+
+    if type(conditions) is not list:
+        conditions = [conditions]
+
+    for condition in conditions:
+        if type(condition) is not dict:
+            if attribute['Value'] == condition:
+                result = True
```

Comment:
nitpick: instead of setting/overwriting the `result` variable, we could also return `False` if any of the tests fails, and then `return True` at the end of the function (which means that all tests have passed)